packages/devtools_app/lib/src/shared/config_specific/server/_server_web.dart

```diff
@@ -402,6 +402,25 @@ Future<ExtensionEnabledState> extensionEnabledState({
   return ExtensionEnabledState.error;
 }
 
+Future<List<String>> requestAndroidBuildVariants(String path) async {
```

Comment:
I can't find a way to reasonably test this function.

If we really want to test it, we need a way to mock  `isDevToolsServerAvailable`.
also will the ci run the test with web platform?

Reply:
I don't think we have a way to test this currently. At first, I thought we might be able to test this with an integration test, but I'm not 100% confident that we start the server for those either 🤔